avocado/plugins/multiplex.py

```diff
@@ -16,11 +16,26 @@
 import sys
 
 from avocado.core import exit_codes
-from avocado.core import tree
 from avocado.core.plugin_interfaces import CLICmd
 from avocado.core.settings import settings
 
 
+_VERBOSITY_LEVELS = {"none": 0, "brief": 1, "normal": 2, "verbose": 3,
+                     "full": 4, "max": 99}
+
+
+def map_verbosity_level(level):
+    if level.isdigit():
+        return int(level)
+    level = level.lower()
+    if level in _VERBOSITY_LEVELS:
+        return _VERBOSITY_LEVELS[level]
+    else:
+        sys.stderr.write("%s is not valid verbosity. ($int, %s)"
```

Comment:
Thinking of users, it's better to give long (but clear) messages than the opposite.  Thanks for addressing this.